WcaOnRails/config/locales/en.yml

```diff
@@ -899,11 +923,29 @@ en:
         one: "WCA Delegate"
         other: "WCA Delegates"
       contact: "Contact"
-      competitor_limit: "Competitor Limit"
-      entry_fee: "Entry Fee"
       information: "Information"
       events: "Events"
       registration_requirements: "Registration requirements"
+      create_wca_account: "Create a WCA account if you don't have one."
+      claim_wca_id: "If this is not your first competition, associate your WCA ID to your WCA account."
+      competitor_limit: "There is a competitor limit of %{competitor_limit} competitors."
+      no_competitor_limit: "There isn't a competitor limit."
+      #context: The space at the end is necessary
+      entry_fee: "The registration fee for this competition is "
```

Comment:
This won't necessarily work for all languages. The right way to do this is to use string interpolation, as you did above with `competitor_limit`. This will also allow you to get rid of the "The space at the end is necessary" comment.

Reply:
I had problems with displaying the amount and the currency correctly, but I'll work on doing it right.